types/d3-dispatch/index.d.ts

```diff
@@ -1,18 +1,74 @@
 // Type definitions for D3JS d3-dispatch module 1.0
 // Project: https://github.com/d3/d3-dispatch/
-// Definitions by: Tom Wanzek <https://github.com/tomwanzek>, Alex Ford <https://github.com/gustavderdrache>, Boris Yankov <https://github.com/borisyankov>
+// Definitions by: Tom Wanzek <https://github.com/tomwanzek>
+//                 Alex Ford <https://github.com/gustavderdrache>
+//                 Boris Yankov <https://github.com/borisyankov>
+//                 denisname <https://github.com/denisname>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-// Last module patch version validated against: 1.0.1
+// Last module patch version validated against: 1.0.3
 
 export interface Dispatch<T extends EventTarget> {
+    /**
+     * Like `function.apply`, invokes each registered callback for the specified type,
+     * passing the callback the specified arguments, with that as the this context.
```

Comment:
This is a light cosmetic note, and one I don't consider a blocker: I might prefer rephrasing this line of the comment like so: ``[...], with `that` as the `this` context.`` It makes it more obvious which parts of the sentence refer to code concerns.